projects/zetasql/project.yaml

```diff
@@ -7,3 +7,4 @@ auto_ccs:
 sanitizers:
```

Comment:
Can we just delete the sanitizer section entirely now that ubsan is working?

Reply:
Deleting sanitizer disables MSAN as it's not turned on by default.

Reply:
Yes right. But msan build is failing, can you check the failure.